zscaler_connector.py

```diff
@@ -793,39 +874,44 @@ def finalize(self):
 
 if __name__ == '__main__':
 
-    import pudb
     import argparse
+
+    import pudb
     pudb.set_trace()
 
     argparser = argparse.ArgumentParser()
 
     argparser.add_argument('input_test_json', help='Input Test JSON file')
     argparser.add_argument('-u', '--username', help='username', required=False)
     argparser.add_argument('-p', '--password', help='password', required=False)
+    argparser.add_argument('-v', '--verify', action='store_true', help='verify', required=False, default=False)
 
     args = argparser.parse_args()
+    verify = args.verify
     session_id = None
 
     if (args.username and args.password):
         login_url = BaseConnector._get_phantom_base_url() + "login"
         try:
             print("Accessing the Login page")
-            r = requests.get(login_url, verify=False)
+            r = requests.get(    # nosemgrep: python.requests.best-practice.use-timeout.use-timeout
+                login_url, verify=verify)
             csrftoken = r.cookies['csrftoken']
             data = {'username': args.username, 'password': args.password, 'csrfmiddlewaretoken': csrftoken}
             headers = {'Cookie': 'csrftoken={0}'.format(csrftoken), 'Referer': login_url}
 
             print("Logging into Platform to get the session id")
-            r2 = requests.post(login_url, verify=False, data=data, headers=headers)
+            r2 = requests.post(    # nosemgrep: python.requests.best-practice.use-timeout.use-timeout
```

Comment:
remove blank space

Reply:
I removed 2 additional spaces before inline comment. Now there are only 2 spaces left that are necessary to be added before an inline comment.